### DISCUSSION TITLE
en/chapter_sorting/bubble_sort/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1398] [Announcements]
# en/chapter_sorting/bubble_sort/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_sorting/bubble_sort/

**Comment by jv01-27** [comment]
I love how everything is so well organized and explained. I used this algorithm in an evaluation for interns.

**Reply by krahets** [reply]
Appreciated!
